torchprime/launcher/cli.py

```diff
@@ -194,6 +194,57 @@ def create_and_activate_gcloud(gcloud_config_name, config: Config):
   )
 
 
+@cli.command(
+  context_settings=dict(
+    ignore_unknown_options=True,
+  )
+)
+@click.argument("args", nargs=-1, type=click.UNPROCESSED)
+@click.option("--use-hf", is_flag=True, help="Use HuggingFace transformer")
+def dbrun(args, use_hf: bool):
```

Comment:
It's unclear what "dbrun" stands for. Perhaps we could write

```
@cli.command(
  name="docker-run",
  context_settings=dict(
    ignore_unknown_options=True,
  )
)
```

Which will make the command "tp docker-run ..."